eRecruiter.Api/Responses/Applicant/ApplicantHistoryResponse.cs

```diff
@@ -0,0 +1,8 @@
+﻿using eRecruiter.Api.Parameters;
+
+namespace eRecruiter.Api.Responses
+{
+    public class ApplicantHistoryResponse : ApplicantHistoryParameter
```

Comment:
Why are we returing the same object that we sent? Wouldn't it be enough to return a status code ``201 Created`` or a simple status object?